### ISSUE TITLE
Cannot GET /

### ISSUE THREAD

**Issue by jkleiser** [yysun/apprun#126]
I just made a new app by doing `npx apprun --init --spa`. When I try to run it with `npm start`, I get `Cannot GET /` in my browser. Has that got something to do with this message I got while installing:
```
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
```
I'm using node v16.13.1 and npm v8.1.2.

**Comment by yysun** [comment]
It was because the webpack dev server serves static content from the _public_ folder now. Please update the _webpack.config.js_ by adding the **static** option.

```js
  devServer: {
    open: true,
    static: path.join(__dirname),
  },
```
I will update the CLI to support the _public_ folder.

**Comment by jkleiser** [comment]
Thanks a lot. I went for this other variant instead: `npx apprun --init --esbuild`. It gives me a simpler app, but maybe that's all I need for now.
It's quite some time now since I started on a new project. It feels good. I'm curious to see how it turns out.

**Comment by yysun** [comment]
Thank you for your long-time support!

**Comment by yysun** [comment]
Fixed in 3.28.8
